oauth2_http/java/com/google/auth/http/AuthHttpConstants.java

```diff
@@ -0,0 +1,18 @@
+package com.google.auth.http;
+
+import java.util.regex.Pattern;
+
+/**
+ * Constants used for auth in http
+ */
+public class AuthHttpConstants {
```

Comment:
In general, we should keep public surface to a minimum, since it needs to be documented and impacts the overall digestibility and discover-ability.